src/services/synapse/produceSynapseUserId.ts

```diff
@@ -7,10 +8,31 @@ const serverName = process.env.SYNAPSE_SERVER_NAME;
  * @param skipPrePostfix If true, the @ and :serverName will not be added
  * @returns Synapse user id
  */
-export function produceSynapseUserId(
+export async function produceSynapseUserId(
   member: ProposalUser,
+  synapseService?: SynapseService,
   skipPrePostfix: boolean = false
-): string {
+): Promise<string> {
```

Comment:
Initially i tried to to retrieve it from container, but then it throw errors. It's probably because produceSynapseuserID function itself is being used in the service class